version.gradle

```diff
@@ -1,3 +1,3 @@
 ext {
-  PROSPECTIVE_RELEASE_VERSION = '2.1.1'
+  PROSPECTIVE_RELEASE_VERSION = '2.1.2'
```

Comment:
2.1.1 hasn't been released... only 2.1.0! so the version can stay

Reply:
Or actually... the prospective version will likely contain the new Squash feature, so it'll be numbered 2.2.0, not 2.1.1 (lemme assign the milestone)